db/controllers/controllers.js

```diff
@@ -55,3 +56,18 @@ exports.getAllArticles = (req,res,next)=>{
 }
 
 
+exports.patchArticleById =(req,res,next)=>{
+    const patchData=req.body;
+    const {article_id}=req.params;
+    const promises=[updateArticleById(patchData.inc_votes,article_id),fetchArticleById(article_id)];
```

Comment:
Do we need to check for the article existing before attempting to post a comment? The insert comment query will be violating a foreign key restraint if the article doesn't exist so we can rely on psql to throw an error here.

Reply:
Okay, thanks, I have updated the code (and will address the same issue in a different task!)